test/box-luatest/error_subsystem_improvements_test.lua

```diff
@@ -1,6 +1,22 @@
 local t = require('luatest')
 local g = t.group()
 
+-- Test the `prev' argument to the table constructor of `box.error.new'
+-- (gh-9103).
+g.test_error_new_prev_arg = function()
+    local e1 = box.error.new({"errare"})
+    local e2 = box.error.new({"humanum", prev = e1})
+    local _, e3 = pcall(box.error, {"est", prev = e2})
```

Comment:
Why `pcall`?

Reply:
To test `box.error()`, not only `box.error.new()`.